MetadataDevelopment/ImplementationChecks/EBMLRegistry.xml

```diff
@@ -106,29 +125,159 @@
     <target_format_part></target_format_part>
     <citation></citation>
     <definition>Ensure MKV Element ID is registered in specdata.xml (as of Dec. 13, 2014 this is 224 registered Element IDs)</definition>
+    <reference_file>unknown_element.mkv</reference_file>
+    <note>deprecated by EBML-ELEMENT-VALID-PARENT</note>
   </rule>
   <rule icid="EBML-ELEM-SIZE-7F">
     <name>Element Size 0x7F Reservation</name>
     <authority>EBML Specification</authority>
     <target_format>EBML</target_format>
     <target_format_part>EBML Element Size</target_format_part>
-    <citation>http://matroska.org/technical/specs/rfc/index.html</citation>
+    <citation></citation>
     <definition>"Note that the shortest encoding form for 127 is 0x407f since 0x7f is reserved." If Element Size is set to 0x11111111 but element size is actually 127 bytes provide a warning.</definition>
```

Comment:
For readability, either:

- add a comma after "127 bytes"
- put "Provide a warning" before "if Element Size"